### PR TITLE
New Getting started experience

### PR DESCRIPTION
**Pull request by microsoftsam**
screen shot 2017-03-27 at 4 55 57 pm
screen shot 2017-03-27 at 4 50 12 pm
screen shot 2017-03-27 at 4 50 06 pm
screen shot 2017-03-27 at 4 50 01 pm
screen shot 2017-03-27 at 4 49 51 pm

Note: No changes to the authoring of the getting started guide, the first element is always rendered in the header, the rest is rendered in the hint.

### CODE REVIEW

File: pxtsim/embed.ts [hunk 1]
```diff
@@ -100,7 +100,8 @@ namespace pxsim {
         subtype: "steploaded";
         data: {[index: string]: number };
         showCategories?: boolean;
-        location?: string;
+        headercontent: string;
+        content: string;
```
Comment (pelikhan) [inline pxtsim/embed.ts]: optimal?

File: webapp/src/app.tsx [hunk 2]
```diff
@@ -1336,10 +1347,15 @@ ${compileService ? `<p>${lf("{0} version:", "C++ runtime")} <a href="${Util.html
                     this.newProject();
                 }
             }).finally(() => {
-                this.setState({ active: true, tutorial: null, tutorialName: null, tutorialSteps: null, tutorialStep: -1 });
+                this.setState({ active: true, tutorialOptions: null });
             });
     }
 
+    showTutorialHint() {
```
Comment (pelikhan) [inline webapp/src/app.tsx]: Add tick

Comment (microsoftsam) [inline webapp/src/app.tsx]: tick in tutorial.tsx

Comment (microsoftsam) [inline webapp/src/app.tsx]: moving to app.tsx

File: pxtsim/embed.ts (hunk 1 above)
Comment (microsoftsam) [inline pxtsim/embed.ts]: The rendering goes on in the iframe and the main frame just displays the html. what's your proposition?
Bear in mind we're trying not to include the runner in the main editor.

**Pull request merged**